src/lasdi/latent_space.py

```diff
@@ -118,7 +151,13 @@ def encoder(self, x):
 
         for i in range(1, self.n_layers + 1):
             fc = getattr(self, 'fc' + str(i) + '_e')
-            x = self.g_e(fc(x))
+            x = fc(x) # apply linear layer
+            if hasattr(self, 'a1'): # test if there is at least one attention layer
```

Comment:
@Eisenl10 , can we make a function that takes `x` and index `i` for this if statement, so that it can be also used in line 174-177? I believe it would be less error-prone for future development.